src/patternfly/components/Label/label--variants.hbs

```diff
@@ -2,37 +2,58 @@
   {{> label
       label--id=(concat label--variants--id '-status-default')
       label-text--value=(concat label--variants--title)}}
+
   {{> label
       label--IsRemovable=true
       label--id=(concat label--variants--id '-status-removable')
       label-text--value=(concat label--variants--title ' removable')}}
+
   {{> label
       label-content--IsLink=true
       label--id=(concat label--variants--id '-status-link')
       label-text--value=(concat label--variants--title ' link')}}
+
   {{> label
       label-content--IsButton=true
       label--id=(concat label--variants--id '-status-button')
       label-text--value=(concat label--variants--title ' button')}}
+
   {{> label
       label--IsRemovable=true
       label-content--IsLink=true
       label--id=(concat label--variants--id '-status-link-removable')
       label-text--value=(concat label--variants--title ' link removable')}}
+
   {{> label
       label--IsRemovable=true
       label-content--IsButton=true
       label--id=(concat label--variants--id '-status-button-removable')
       label-text--value=(concat label--variants--title ' button removable')}}
+
   {{> label
       label--id=(concat label--variants--id '-status-truncated')
       label-text--value=(concat label--variants--title ', max-width truncation customization')
       label-text--attribute='style="--pf-v5-c-label__text--MaxWidth: 28ch"'}}
+
   {{> label
       label--IsRemovable=true
       label--id=(concat label--variants--id '-status-truncated-removable')
       label-text--value=(concat label--variants--title ' removable, max-width truncation customization')
       label-text--attribute='style="--pf-v5-c-label__text--MaxWidth: 28ch"'}}
+
+  {{> label
+      label--IsRemovable=true
+      label--IsDisabled=true
+      label-content--IsLink=true
+      label--id=(concat label--variants--id '-status-link-removable-disabled')
+      label-text--value=(concat label--variants--title ' link removable (disabled)')}}
```

Comment:
I don't think we should show link labels being disabled (at least not the whole label). While you can't click the link to activate it, you can still activate/focus it via keyboard. Also not sure when it may make sense to disable an `<a>` element, and at that point it's not really a link but rather just text.

If this variant could be updated so that only the "remove" button is disabled, but the rest of the label has its normal styling (and the link works on mouse click) it could be worth keeping in, but otherwise probably better to just remove it.

Reply:
Great feedback! Just for reference from our convo offline, adding for now since we have this as an established (anti-?)pattern already in the library, and will investigate a better approach to this via [the spike issue you created](https://github.com/patternfly/patternfly/issues/6448). If we can fit this change in the timeline for penta, that would be great!